pkg/util/scrubber/default.go

```diff
@@ -116,25 +116,27 @@ func matchCert() *regexp.Regexp {
 	)
 }
 
+// matchYAMLKeyWithListValue matches YAML keys with list values.
```

Comment:
Let's add a note here that this only works for lists of non-array values.  For example, it won't correctly match

```yaml
key: [
  [a, b, c],
  def]
```
I think that's OK for current use-cases, but let's be sure to mention it.

Reply:
Good point, added a note here: https://github.com/DataDog/datadog-agent/pull/10613/commits/e977af5c2ceb2e26721725a1564ddc297cfbca7f